OmiseSDK/CardNumber.swift

```diff
@@ -15,7 +15,11 @@ public final class CardNumber {
             .first
     }
     
-    public static func format(pan: String) -> String {
+    @objc(brandForPan:) public static func __brand(pan: String) -> Int {
```

Comment:
I think `NSNotFound` is 🆗 for this since the method is actually "finding" a brand in a sense.